### PR TITLE
Feat: view login

### PR DESCRIPTION
**Pull request by Kender-Mendoza**
- added Create user view
- added login view

### CODE REVIEW

File: meter-logger/Features/Login/ViewModel/authViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    }
+    
+    func showMessage(isSuccessMessage: Bool, body: String) {
+        self.messageTitle = isSuccessMessage ? NSLocalizedString("success", comment: "") : NSLocalizedString("error", comment: "")
+        self.messageBody = body
+        self.showMessage = true
+    }
+
+    private func refreshToken() {
+        print("si entro a refrescarlo")
```
Comment (Orivera04) [inline meter-logger/Features/Login/ViewModel/authViewModel.swift]: Quita esto

File: meter-logger/ContentView.swift [hunk 2]
```diff
@@ -9,13 +10,16 @@ import SwiftUI
...
     @State var presentSideMenu = false
+    @EnvironmentObject var authViewModel: AuthViewModel
 
     var body: some View {
-       NavbarView(presentSideMenu: $presentSideMenu) {
-           NavigationStack {
-               HomeView()
-           }
-       }
+        if authViewModel.userHaveSession  {
```
Comment (Orivera04) [inline meter-logger/ContentView.swift]: userHaveSession

userHasSession?

File: meter-logger/Features/Login/ViewModel/authViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+//  MeterLogger
+//
+//  Created by Kender Esteban Mendoza on 29/2/24.
+//
+
+import Foundation
+
+class AuthViewModel: ObservableObject {
+    @Published var user: User
+    @Published var userHaveSession: Bool = false
```
Comment (Orivera04) [inline meter-logger/Features/Login/ViewModel/authViewModel.swift]: UserHasSession

File: meter-logger/Shared/Helpers/AuthTokenManager.swift [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        }
+        
+        return false
+    }
+    
+    private func setPayloadDecoded() {
+        self.payloadDecoded = self.decoddingPayloadToken()
+    }
+            
+    private func decoddingPayloadToken() -> [String: Any] {
```
Comment (Orivera04) [inline meter-logger/Shared/Helpers/AuthTokenManager.swift]: decoddingPayloadToken

decoding ?

**Review by Orivera04 — CHANGES_REQUESTED** [review]
Par de comentarios

**Review by oscar-rivera-skydropx — APPROVED** [review]

**Pull request merged**